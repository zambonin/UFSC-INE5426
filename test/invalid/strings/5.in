# improper arithmethics between character variables
char a, b
a = 's'
b = 'd'
a = a + b
