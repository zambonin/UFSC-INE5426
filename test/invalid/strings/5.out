[Line 6] semantic error: attribution operation expected character but received character array
